### PR TITLE
Consolidate header: design language

### PR DESCRIPTION
**Pull request by wesoudshoorn**
# Feedback
- [x] Update submodule to reflect the active links

### CODE REVIEW

**Comment by matsimitsu** [comment]
The contrast for an "active" top-bar menu link is not very high, from a small distance, I can't see what link is active (it's pricing in this case)

affordable_request_based_pricing___appsignal

[edit] 

Odd.. on blog with the latest submodule update it looks different:

appsignal_blog__the_latest_on_everything_appsignal

File: components/_tag.sass [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// generic styling
```
Comment (tombruijn) [inline components/_tag.sass]: we can remove this comment, I think?

File: components/_tag.sass [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  @each $color in $colors
+    &.#{nth($color, 1)}
+      background: #{nth($color, 2)}
+      color: #fff
+
+  &.white
+    color: $dark
+    background: #fff
+
```
Comment (tombruijn) [inline components/_tag.sass]: unnecessary empty line

File: modules/_header.sass [hunk 3]
```diff
@@ -49,11 +49,15 @@ $toggle-width: 900px
         display: flex
         width: 100%
 
-        > li
-          padding: 15px
+        >li
```
Comment (tombruijn) [inline modules/_header.sass]: inconsistent usage of the `> tag` selector. The rest of the file includes a space after the `>` symbol

**Comment by tombruijn** [comment]
test slack integration (will remove comment)

**Review by jeffkreeftmeijer — APPROVED** [review]
Checked in docs, blog and -server, looks good there. Code looks good.

**Pull request merged**